Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM python:2.7
+
+# Configure the working directory
+RUN mkdir -p /opt/project
+WORKDIR /opt/project
+
+# Airflow version upgrade to 1.10
+ENV SLUGIFY_USES_TEXT_UNIDECODE=yes
+
+# Install and update pip
+# Pin the version because pip>=10.0 does not support the --download flag  which is required for dataflow
+RUN pip install -U --ignore-installed pip==9.0.3
+
+# Download and install google cloud. See the dockerfile at
+# https://hub.docker.com/r/google/cloud-sdk/~/dockerfile/
+RUN  \
+  export CLOUD_SDK_APT_DEPS="curl gcc python-dev python-setuptools apt-transport-https lsb-release openssh-client git" && \
+  export CLOUD_SDK_PIP_DEPS="crcmod" && \
+  apt-get -y update && \
+  apt-get install -y $CLOUD_SDK_APT_DEPS && \
+  pip install -U $CLOUD_SDK_PIP_DEPS && \
+  export CLOUD_SDK_VERSION="198.0.0" && \
```

Comment:
@smpiano you are using an old version of the SDK. Check pipe-vms-chile for the Dockerfile. move the version to the upper part of the file.